app/views/posts/index.html.erb

```diff
@@ -18,7 +18,8 @@
             <%= hidden_field_tag(:user_id, current_user.id) %>
             <%= hidden_field_tag(:post_id, post.id) %>
             <% already_liked = post.likes.any?{ |like| like.user_id == current_user.id } %>
-            <%= submit_tag "Like (#{post.likes.count})", :disabled => already_liked %>
+            <%= submit_tag "Like", :disabled => already_liked, :class => "like_button", :onclick => "return like_count(#{post.id}, #{post.likes.count})" %>
```

Comment:
It's generally preferred to use DOM event binding (e.g. `window.addEventListener`) rather than onclick. Separation of concerns is the reason — we're putting javascript code in our HTML elements and that makes it hard for us to reason about our application and find stuff when we need it.

Have a go and then come over if you need any pointers.